elixir/apps/domain/lib/domain/auth/identity/query.ex

```diff
@@ -4,6 +4,9 @@ defmodule Domain.Auth.Identity.Query do
   def all do
     from(identities in Domain.Auth.Identity, as: :identities)
     |> where([identities: identities], is_nil(identities.deleted_at))
+    |> join(:inner, [identities: identities], actors in assoc(identities, :actor), as: :actors)
+    |> where([actors: actors], is_nil(actors.deleted_at))
+    |> where([actors: actors], is_nil(actors.disabled_at))
```

Comment:
👍 Disabling a user is relatively common in current Firezone, thanks for keeping this around. The most common use case I've seen this pop up is when an admin wants to only temporarily disable access for a particular user.